lib/prompts/selectHubDBTablePrompt.ts

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs';
+import { promptUser } from './promptUtils';
+import { i18n } from '../lang';
+import { logError } from '../errorHandlers/index';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { fetchTables } from '@hubspot/local-dev-lib/api/hubdb';
+import { EXIT_CODES } from '../enums/exitCodes';
+import { Table } from '@hubspot/local-dev-lib/types/Hubdb';
+import { isValidPath, untildify } from '@hubspot/local-dev-lib/path';
+
+const i18nKey = 'lib.prompts.selectHubDBTablePrompt';
+
+async function fetchHubDBOptions(accountId: number) {
+  try {
+    const {
+      data: { results: tables },
+    } = await fetchTables(accountId);
+    if (tables.length === 0) {
+      logger.log(i18n(`${i18nKey}.errors.noTables`, { accountId }));
+      return;
+    }
+    if (tables.every(table => table.rowCount === 0)) {
+      logger.error(i18n(`${i18nKey}.errors.allTablesEmpty`, { accountId }));
+      process.exit(EXIT_CODES.ERROR);
+    }
+    return tables;
+  } catch (error) {
+    logError(error, { accountId });
+    logger.error(i18n(`${i18nKey}.errors.errorFetchingTables`, { accountId }));
```

Comment:
Will this log two separate errors to the console?

Reply:
Yes, it will. I wanted to add some context for the user that we were unable to fetch tables from their account apart from whatever the actual error is. Any suggestions for different formatting?

Reply:
Ok thanks for the clarification. I think you could probably just keep the bottom (user-friendly) error and then switch the other one to a debug log for debugging.